### DISCUSSION TITLE
How can I customize my default Jekyll theme to my preferred layout or design?

### DISCUSSION THREAD

**Discussion by osioke** [community/community#21543] [Other Feature Feedback, Questions, & Ideas]
<p>The basic themes available for Jekyll are a bit uninteresting and only meant to display projects. While this might be okay for some, it isn’t okay for me as I want to use Pages for my personal portfolio site. Any ideas?</p>

**Comment by AlMcKinlay** [comment, marked as answer]
<p>Hi there,</p>
<p>You can add custom css and html files into your Jekyll site.</p>
<p>From <a href="https://help.github.com/articles/customizing-css-and-html-in-your-jekyll-theme/" rel="nofollow noopener">this page on github help</a>, you can create a scss/css file at <em>/assets/css/style.scss</em> and add the following to the top of the file:</p>
<pre><code>---
---

@import "{{ site.theme }}";
</code></pre>
<p>Then any custom css can be put after these lines in this file and it will be included in your site.</p>

**Comment by osioke** [comment]
<p>Awesome!</p>
<p>That was really helpful. Thank you for saving me time from scouring through so many help articles :smiley: </p>

**Comment by DirtyF** [comment]
<p>Check out  this new Jekyll plugin by <a class="mention" href="/benbalter">@benbalter</a> <a href="https://github.com/benbalter/jekyll-remote-theme" rel="nofollow noopener">https://github.com/benbalter/jekyll-remote-theme</a>, not yet supported by GitHub Pages.</p>

**Comment by osioke** [comment]
<p>Thank you <a class="mention" href="/dirtyf">@dirtyf</a></p>

**Comment by DirtyF** [comment]
<p>Turns out <strong>jekyll-remote-theme</strong> is now whitelisted on GitHub Pages, so you can use any GitHub hosted theme with Jekyll.</p>

**Comment by osioke** [comment]
<p>Oh, I will definitely check that out. Thanks!</p>
